src/metabase/models/serialization.clj

```diff
@@ -1,3 +1,4 @@
+;; -*- outline-regexp: "[; ]+#+[[:space:]]+" -*-
```

Comment:
Did you mean to leave this here? Never seen its ilk before :-)

Reply:
Yeah, trying to make this file a bit more useful for emacs 😁 I can put this down but... hopefully it's not too bad :)